validator/lib/validatePnts.js

```diff
@@ -181,6 +183,16 @@ function validatePnts(content) {
         return 'Feature table property BATCH_LENGTH must be less than or equal to POINTS_LENGTH.';
     }
 
+    if (defined(featureTableJson.BATCH_ID)) {
+        var featureTable = new Cesium3DTileFeatureTable(featureTableJson, featureTableBinary);
+        var batchIds = featureTable.getPropertyArray('BATCH_ID', ComponentDatatype.UNSIGNED_SHORT, 1);
```

Comment:
The `BATCH_ID` property is the one exception where it may declare a component type other than the default of `UNSIGNED_SHORT`.

Cesium has the same mistake which I'll address. The fixed code might look like:

```
var componentDatatype = ComponentDatatype.fromName(defaultValue(featureTableJson.BATCH_ID.componentType, 'UNSIGNED_SHORT'));
var batchIds = featureTable.getPropertyArray('BATCH_ID', componentDatatype, 1);